tests/test_keep_shutter_open.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+from typing import Iterable
+
+import pytest
+
+import useq
+from useq import AxesBasedAF, MDASequence, ShutterOpenAxes
+
+ZRANGE2 = useq.ZRangeAround(range=2, step=1)
```

Comment:
why do I get the feeling that you just copied and pasted everything from the autofocus one?  :)

this is wayyy to much.  all you're really testing here is that `should_keep_open` works as it should. and all it's testing is that if a "watched" axis changes you return True.  we probably didn't need all these cases for autofocus in the first place after all the logic was cleaned up, and we shouldn't continue to propogate it.